src/javascript-node-mongo/devcontainer-template.json

```diff
@@ -11,6 +11,7 @@
             "type": "string",
```

Comment:
Can you update the default L20 to "22-bookworm" as well?

Reply:
done